mlir/utils/jenkins/Jenkinsfile.nightly-perf

```diff
@@ -48,15 +48,17 @@ pipeline {
                     cmakeArgs: '''-DMLIR_MIOPEN_DRIVER_ENABLED=1'''
             }
         }
+        stage('Install PrettyTable (temporary)') {
+            sh 'pip3 install PrettyTable'
+        }
         stage('Performance Test') {
             steps {
-                sh '''
-                    cd build
-                    cmake --build . --target MIOpenDriver.py
-                    pip3 install PrettyTable
-                    # Run MLIR perf benchmarks.
-                    python3 ./llvm/MIOpenDriver.py
-                '''
+                cmake arguments: '--build . MIOpenDriver.py',\
+                    installation: 'InSearchPath', workingDir: 'build'
+                dir('build') {
+                    // Run MLIR perf benchmarks
+                    sh 'python3 ./llvm/MIOpenDriver.py'
+                }
```

Comment:
You might want to run this Jenkins job to make sure it works well. I did try dir('build') in my development branch, but for some reason the publishHTML step crashed, though MLIR_vs_MIOpen.html was generated correctly. I don't know why. I'm not very familiar with publishhtml plugins. So I decided to use shell script.